test/ArrayField_test.js

```diff
@@ -38,21 +38,21 @@ describe("ArrayField", () => {
       expect(legend.id).eql("root__title");
     });
 
-    it("should contain no field in the list by default", () => {
+    it("should contain no field in the list by default", function*() {
```

Comment:
Could you elaborate a little bit on the conversion to generator functions? To take one example at random, I don't understand why this test needed to be converted. It's not yielding anything, nor even (so far as I can tell) causing any asynchronous functions to be invoked.